src/DependencyInjection/LazyContainerFactory.php

```diff
@@ -604,9 +603,10 @@ private function registerTagged(Container $container, array $classes, string $ta
         }
     }
 
-    private function createPHPStanServices(LazyRectorConfig $container): void
+    private function createPHPStanServices(LazyRectorConfig $lazyRectorConfig): void
     {
-        $container->singleton(
+        $container = null;
+        $lazyRectorConfig->singleton(
```

Comment:
looks like a bug was introduced here

Reply:
Yes, I tested `ReturnTypeFromStrictParamRector` to use `ClassMethodReturnTypeOverrideGuard` instead as it change return type https://github.com/rectorphp/rector-src/pull/4703/files#diff-a5951960805d541e1e180ef2e37996275c1d53f8f6d3186435fce3b14b1727d6

Reply:
@TomasVotruba it seems it related to @TomasVotruba change on PR https://github.com/rectorphp/rector-src/actions/runs/5784912801/job/15676537339

Reply:
It crashes when variable is renamed and deafult null is added at the same time. It colision of 2 rules, but we can skip this manually now.